### PR TITLE
You can set a hair gradient style in the character creation menu and colorist quirk that lets you spawn with hair dye

### PR DESCRIPTION
**Pull request by Chubbygummibear**
## About The Pull Request
Ports the port https://github.com/yogstation13/Yogstation/pull/12488

image
image

## Why It's Good For The Game

more character creation options good, especially in this case where it was something you could do in game, but not until you acquired hair dye

## Changelog
:cl: @nmajask, Truzkawa, Chubbygummibear
add: Hair Gradient style and color are part of character appearance menu now
add: new neutral quirk colorist that lets you spawn with a bottle of hair dye spray
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
AAAAAAAAAAAAAAAAAAAAAA THANK YOU!!

**Comment by AtlasRules** [comment]
based... so bzvsased.......

File: code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -120,6 +120,30 @@
...
+	)
+	where = H.equip_in_one_of_slots(spraycan, slots, FALSE) || "at your feet"
+
+/datum/quirk/colorist/post_add()
+	if(where == "in your backpack")
+		var/mob/living/carbon/human/H = quirk_holder
+		SEND_SIGNAL(H.back, COMSIG_TRY_STORAGE_SHOW, H)
+
+	to_chat(quirk_holder, "<span class='notice'>Your bottle of hair dye spray is [where].</span>")
+
```
Comment (MarkSuckerberg) [inline code/datums/traits/neutral.dm]: Won't need this, it's already in the loadout, rest is 👌 though

Comment (Chubbygummibear) [reply code/datums/traits/neutral.dm]: completely forgot about the loadout lmao. i'm so used to that not being a thing from yog. quirk gone now 👍

**Pull request merged**